config/core/configmaps/defaults.yaml

```diff
@@ -81,3 +81,8 @@ data:
     # disables this throttling and lets through as many requests as
     # the pod receives.
     container-concurrency: "0"
+
+    # pod-autoscaler-class specifies the default pod autoscaler class
+    # that should be used if none is specified. If omitted, the
+    # Knative Horizontal Pod Autoscaler (KPA) is used by default.
+    pod-autoscaler-class: "kpa.autoscaling.knative.dev"
```

Comment:
So if I am reading the logic correctly if the user passes in an invalid value to this class we will fall back to using the KPA? This might seem like the configmap value is not taking place.

Reply:
If _omitted_ the KPA is used, if _invalid_ the exact same thing happens -- unless I missed something -- as if the user set an invalid value directly when creating the service.